plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/admin/LogoutsCommand.java

```diff
@@ -0,0 +1,37 @@
+package net.dzikoysk.funnyguilds.feature.command.admin;
+
+import net.dzikoysk.funnycommands.stereotypes.FunnyCommand;
+import net.dzikoysk.funnyguilds.feature.command.AbstractFunnyCommand;
+import net.dzikoysk.funnyguilds.feature.command.UserValidation;
+import net.dzikoysk.funnyguilds.shared.FunnyFormatter;
+import net.dzikoysk.funnyguilds.user.User;
+import org.bukkit.command.CommandSender;
+
+import static net.dzikoysk.funnyguilds.feature.command.DefaultValidation.when;
+
+public final class LogoutsCommand extends AbstractFunnyCommand {
+
+    @FunnyCommand(
+            name = "${admin.logouts.name}",
+            permission = "funnyguilds.admin",
+            completer = "guilds:3",
+            acceptsExceeded = true
+    )
+    public void execute(CommandSender sender, String[] args) {
+        when(args.length < 1, this.messages.generalNoTagGiven);
+        when(args.length < 2, this.messages.adminNoLogoutsGiven);
+
+        User user = UserValidation.requireUserByName(args[0]);
+
+        int logouts = Integer.parseInt(args[1]);
+
+        user.getRank().setLogouts(logouts);
```

Comment:
Call LogoutsChangeEvent (see how it should look for [deaths command](https://github.com/FunnyGuilds/FunnyGuilds/blob/f33f96560f9fc22c9122373a2b4488fe8f613dfa/plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/admin/DeathsCommand.java#L39))